src/convertToObject.js

```diff
@@ -6,7 +6,22 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const styleObject = {};
+
+  const styleDeclarations = sourceString.split(';');
+
+  styleDeclarations.forEach((style) => {
+    const parts = style.split(':');
+
+    if (parts.length === 2) {
+      const property = parts[0].trim();
+      const value = parts[1].trim();
+
+      styleObject[property] = value;
```

Comment:
Good job checking if there are exactly two parts after the split. However, you should also handle cases where the style declaration is empty or invalid, which might occur if there are consecutive semicolons.

Reply:
Not fixed